app/controllers/admin/order_cycles_controller.rb

```diff
@@ -45,49 +45,38 @@ def new
     def create
       @order_cycle = OrderCycle.new(params[:order_cycle])
 
-      respond_to do |format|
-        if @order_cycle.save
-          OpenFoodNetwork::OrderCycleFormApplicator.new(@order_cycle, spree_current_user).go!
-          invoke_callbacks(:create, :after)
-          flash[:notice] = I18n.t(:order_cycles_create_notice)
-          format.html { redirect_to admin_order_cycles_path }
-          format.json { render :json => { success: true } }
-        else
-          format.html
```

Comment:
Can you explain why HTML is unnecessary? We never used it? Or can you reference to a commit when we stopped using it, but didn't change this method?

Reply:
As far as I know we never used it.

Rohan's original commit that added this action used json from the very beginning: aacc36ea440d7b84807d5a3b6e6b27a0d3c116d5